ble/FunctionPointerWithContext.h

```diff
@@ -83,6 +95,31 @@ class FunctionPointerWithContext {
     }
 
     /**
+     * @brief Same as above
+     */
+    void operator()(ContextType context) const {
+        call(context);
+    }
+
+    /** Same as above, workaround for mbed os FunctionPointer implementation. */
+    void call(ContextType context) {
```

Comment:
it would be surprising to have call() behave differently depending on constness of the functionpointer object. Why do you need this?